### PR TITLE
Fix typos / add comments for integrations

### PR DESCRIPTION
**Pull request by ribice**
Fixing typos and adding some comments ...

### CODE REVIEW

File: echo/sentryecho.go [hunk 1]
```diff
@@ -10,11 +10,19 @@ import (
...
 
-// The identifier of the Echo SDK.
-const sdkIdentifier = "sentry.go.echo"
+const (
+	// sdkIdentifier is the identifier of the Echo SDK.
+	sdkIdentifier = "sentry.go.echo"
 
-const valuesKey = "sentry"
-const transactionKey = "sentry_transaction"
+	// valuesKey is used as a key to store the Sentry Hub instance in echo.Context.
```
Comment (cleptric) [inline echo/sentryecho.go:17]:
```suggestion
	// valuesKey is used as a key to store the Sentry Hub instance on the  echo.Context.
```

File: echo/sentryecho.go [hunk 2]
```diff
@@ -10,11 +10,19 @@ import (
...
+	valuesKey = "sentry"
+
+	// transactionKey is used as a key to store the Sentry transaction in echo.Context.
```
Comment (cleptric) [inline echo/sentryecho.go:20]:
```suggestion
	// transactionKey is used as a key to store the Sentry transaction on the echo.Context.
```

File: fiber/sentryfiber.go [hunk 3]
```diff
@@ -29,28 +34,26 @@ type handler struct {
...
-	if options.Timeout != 0 {
-		handler.timeout = options.Timeout
-	}
-
-	return handler.handle
+	return (&handler{
+		repanic:         options.Repanic,
+		timeout:         options.Timeout,
+		waitForDelivery: options.WaitForDelivery,
+	}).handle
```
Comment (cleptric) [inline fiber/sentryfiber.go:56]: This reduces readlbility

File: gin/sentrygin.go [hunk 4]
```diff
@@ -13,11 +13,16 @@ import (
...
 
-// The identifier of the Gin SDK.
-const sdkIdentifier = "sentry.go.gin"
+const (
+	// sdkIdentifier is the identifier of the Gin SDK.
+	sdkIdentifier = "sentry.go.gin"
 
-const valuesKey = "sentry"
-const transactionKey = "sentry_transaction"
+	// valuesKey is used as a key to store the Sentry Hub instance in gin.Context.
```
Comment (cleptric) [inline gin/sentrygin.go:20]:
```suggestion
	// valuesKey is used as a key to store the Sentry Hub instance on the gin.Context.
```

File: gin/sentrygin.go [hunk 5]
```diff
@@ -13,11 +13,16 @@ import (
...
+	valuesKey = "sentry"
+
+	// transactionKey is used as a key to store the Sentry transaction in gin.Context.
```
Comment (cleptric) [inline gin/sentrygin.go:23]:
```suggestion
	// transactionKey is used as a key to store the Sentry transaction on the gin.Context.
```

File: iris/sentryiris.go [hunk 6]
```diff
@@ -12,8 +12,13 @@ import (
 
 // The identifier of the Iris SDK.
 const (
-	sdkIdentifier  = "sentry.go.iris"
-	valuesKey      = "sentry"
+	// sdkIdentifier is the identifier of the Iris SDK.
+	sdkIdentifier = "sentry.go.iris"
+
+	// valuesKey is used as a key to store the Sentry Hub instance in iris.Context.
```
Comment (cleptric) [inline iris/sentryiris.go:18]:
```suggestion
	// valuesKey is used as a key to store the Sentry Hub instance on the iris.Context.
```

File: iris/sentryiris.go [hunk 7]
```diff
@@ -12,8 +12,13 @@ import (
...
+	valuesKey = "sentry"
+
+	// transactionKey is used as a key to store the Sentry transaction in iris.Context.
```
Comment (cleptric) [inline iris/sentryiris.go:21]:
```suggestion
	// transactionKey is used as a key to store the Sentry transaction on the iris.Context.
```

File: logrus/logrusentry.go [hunk 8]
```diff
@@ -11,9 +11,13 @@ import (
...
 )
 
-// The identifier of the Logrus SDK.
-const sdkIdentifier = "sentry.go.logrus"
-const name = "logrus"
+const (
+	// sdkIdentifier is the identifier of the Logrus SDK.
+	sdkIdentifier = "sentry.go.logrus"
+
+	// name is the name of the logger
```
Comment (cleptric) [inline logrus/logrusentry.go:18]:
```suggestion
	// the name of the logger
```

File: fasthttp/sentryfasthttp.go [hunk 9]
```diff
@@ -121,6 +126,7 @@ func GetHubFromContext(ctx *fasthttp.RequestCtx) *sentry.Hub {
 	return nil
 }
 
+// SetHubOnContext attaches *sentry.Hub instance to fasthttp.RequestCtx.
```
Comment (cleptric) [inline fasthttp/sentryfasthttp.go:129]:
```suggestion
// SetHubOnContext attaches the *sentry.Hub instance to the fasthttp.RequestCtx.
```

File: fasthttp/sentryfasthttp.go [hunk 10]
```diff
@@ -14,8 +14,13 @@ import (
 )
 
 const (
-	sdkIdentifier  = "sentry.go.fasthttp"
-	valuesKey      = "sentry"
+	// sdkIdentifier is the identifier of the FastHTTP SDK.
+	sdkIdentifier = "sentry.go.fasthttp"
+
+	// valuesKey is used as a key to store the Sentry Hub instance in fasthttp.RequestCtx.
```
Comment (cleptric) [inline fasthttp/sentryfasthttp.go:20]:
```suggestion
	// valuesKey is used as a key to store the Sentry Hub instance on the  fasthttp.RequestCtx.
```

File: fasthttp/sentryfasthttp.go [hunk 11]
```diff
@@ -14,8 +14,13 @@ import (
...
+	valuesKey = "sentry"
+
+	// transactionKey is used as a key to store the Sentry transaction in fasthttp.RequestCtx.
```
Comment (cleptric) [inline fasthttp/sentryfasthttp.go:23]:
```suggestion
	// transactionKey is used as a key to store the Sentry transaction on the fasthttp.RequestCtx.
```

File: fiber/sentryfiber.go [hunk 12]
```diff
@@ -16,8 +16,13 @@ import (
 )
 
 const (
-	valuesKey      = "sentry"
-	sdkIdentifier  = "sentry.go.fiber"
+	// sdkIdentifier is the identifier of the FastHTTP SDK.
+	sdkIdentifier = "sentry.go.fiber"
+
+	// valuesKey is used as a key to store the Sentry Hub instance in fasthttp.RequestCtx.
```
Comment (cleptric) [inline fiber/sentryfiber.go:22]:
```suggestion
	// valuesKey is used as a key to store the Sentry Hub instance on the fasthttp.RequestCtx.
```

File: fiber/sentryfiber.go [hunk 13]
```diff
@@ -16,8 +16,13 @@ import (
...
+	valuesKey = "sentry"
+
+	// transactionKey is used as a key to store the Sentry transaction in fasthttp.RequestCtx.
```
Comment (cleptric) [inline fiber/sentryfiber.go:25]:
```suggestion
	// transactionKey is used as a key to store the Sentry transaction on the fasthttp.RequestCtx.
```

File: fiber/sentryfiber.go [hunk 14]
```diff
@@ -115,17 +118,20 @@ func (h *handler) recoverWithSentry(hub *sentry.Hub, ctx *fiber.Ctx) {
...
 	}
 	return nil
 }
 
+// SetHubOnContext sets the Hub instance on the *fiber.Ctx
 func SetHubOnContext(ctx *fiber.Ctx, hub *sentry.Hub) {
 	ctx.Locals(valuesKey, hub)
 }
 
+// GetSpanFromContext retrieves the Span instance from the *fiber.Ctx
```
Comment (cleptric) [inline fiber/sentryfiber.go:134]:
```suggestion
// GetSpanFromContext retrieves the Span instance from the *fiber.Ctx.
```

File: fiber/sentryfiber.go [hunk 15]
```diff
@@ -115,17 +118,20 @@ func (h *handler) recoverWithSentry(hub *sentry.Hub, ctx *fiber.Ctx) {
...
 
+// GetHubFromContext retrieves the Hub instance from the *fiber.Ctx
 func GetHubFromContext(ctx *fiber.Ctx) *sentry.Hub {
 	if hub, ok := ctx.Locals(valuesKey).(*sentry.Hub); ok {
 		return hub
 	}
 	return nil
 }
 
+// SetHubOnContext sets the Hub instance on the *fiber.Ctx
```
Comment (cleptric) [inline fiber/sentryfiber.go:129]:
```suggestion
// SetHubOnContext sets the Hub instance on the *fiber.Ctx.
```

File: fiber/sentryfiber.go [hunk 16]
```diff
@@ -115,17 +118,20 @@ func (h *handler) recoverWithSentry(hub *sentry.Hub, ctx *fiber.Ctx) {
 	}
 }
 
+// GetHubFromContext retrieves the Hub instance from the *fiber.Ctx
```
Comment (cleptric) [inline fiber/sentryfiber.go:121]:
```suggestion
// GetHubFromContext retrieves the Hub instance from the *fiber.Ctx.
```

File: echo/sentryecho.go [hunk 17]
```diff
@@ -10,11 +10,19 @@ import (
...
+	transactionKey = "sentry_transaction"
+
+	// errorKey is used as a key to store the error in echo.Context.
```
Comment (cleptric) [inline echo/sentryecho.go:23]:
```suggestion
	// errorKey is used as a key to store the error on the echo.Context.
```

**Review by cleptric — APPROVED** [review]

File: fiber/sentryfiber.go (hunk 3 above)
Comment (ribice) [reply fiber/sentryfiber.go:57]: It is the same in Echo, Gin and iris. Just made it consistent.

**Pull request merged**